harness/determined/cli/sso.py

```diff
@@ -121,7 +121,7 @@ def list_providers(args: Namespace) -> None:
     except KeyError:
         raise EnterpriseOnlyError("No SSO providers data")
 
-    if len(sso_providers) == 0:
+    if not sso_providers or len(sso_providers) == 0:
```

Comment:
```python
if not sso_providers:
```
should be sufficient. `bool(<empty_list>)` evaluates to `False`.

Reply:
fixed! thanks